### PR TITLE
OS-2 : Week - 4 : Prescription in appointment history and emergency contact number in doctor's schedule

### PR DESCRIPTION
**Pull request by divyanshkhetan**
## Proposed changes
- Appointments should have prescription in history section.
- In doctor schedule add emergency contact number.

## Types of changes

_Put an `x` in the boxes that apply_

-   [x] Bugfix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] Other (please describe):

## Checklist

_Put an `x` in the boxes that apply_

-   [x] My code follows the [style guidelines of this project](https://docs.google.com/document/d/1GI2Hile8UbGZ82gFe1y4wAVPcNPXip1cmXTzog1S41U/edit)
-   [x] I have performed a self-review of my own code
-   [x] I have created new branch for this pull request
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [x] I have made corresponding changes to the documentation
-   [x] My changes generate no new warnings
-   [x] My changes does not break the current system and it passes all the current test cases.

## Screenshots'
image
image

### CODE REVIEW

**Review by NuthanReddy646 — COMMENTED** [review]
Approve @akshatnema

**Comment by akshatnema** [comment]
@divyanshkhetan Kindly push the migrations of this PR as there is a change in models.py. Review required from @PranshuNayak and @samay-rgb.

**Comment by akshatnema** [comment]
@divyanshkhetan Also, resolve the merge conflicts in the PR.

File: FusionIIIT/applications/health_center/models.py [hunk 1]
```diff
@@ -27,6 +27,7 @@ class Doctor(models.Model):
     doctor_phone = models.CharField(max_length=10)
     specialization = models.CharField(max_length=100)
     active = models.BooleanField(default=True)
+    contact_no = models.IntegerField(default=0)
```
Comment (PranshuNayak) [inline FusionIIIT/applications/health_center/models.py]: use a Charfield instead and keep a max_length and min_length 10

**Review by PranshuNayak — COMMENTED** [review]
Push the migrations for the changes too.

**Comment by divyanshkhetan** [comment]
@PranshuNayak @samay-rgb please take a look.

File: FusionIIIT/applications/health_center/models.py [hunk 2]
```diff
@@ -27,6 +28,7 @@ class Doctor(models.Model):
     doctor_phone = models.CharField(max_length=10)
     specialization = models.CharField(max_length=100)
     active = models.BooleanField(default=True)
+    contact_no = models.CharField(max_length=10, validators=[MinLengthValidator(10)], default=0000000000)
```
Comment (PranshuNayak) [inline FusionIIIT/applications/health_center/models.py]: The default must be a string for a CharField. Use min_length=10 instead of the validator.

Comment (divyanshkhetan) [reply FusionIIIT/applications/health_center/models.py]: @PranshuNayak
That is used for forms.CharField() and not for models.CharField(). That is why I used a validator.

Comment (samay-rgb) [reply FusionIIIT/applications/health_center/models.py]: you need to change the default , it must be string

Comment (divyanshkhetan) [reply FusionIIIT/applications/health_center/models.py]: @samay-rgb PTAL.

Comment (samay-rgb) [reply FusionIIIT/applications/health_center/models.py]: Migration file should be updated too.

**Review by PranshuNayak — COMMENTED** [review]
@divyanshkhetan push new migrations, since you have changed the default value for contact_no

**Comment by divyanshkhetan** [comment]
@PranshuNayak done.

**Review by PranshuNayak — APPROVED** [review]
@akshatnema changes in models.py are fine.

**Review by samay-rgb — APPROVED** [review]

**Pull request merged**